src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -953,11 +957,15 @@ function updateNumScalePossibleValues(questionNumber) {
         $("#maxScaleBox"+idSuffix).val(max);
     }
     
-    if (step < 0.001) {
+    step = roundToThreeDp(step);
+    if (step == 0) {
         step = 0.001;
-        $("#stepBox"+idSuffix).val(step);
     }
-    
+    $("#stepBox" + idSuffix).val(step);        
+    if (isNaN(step)) {
+        $("#stepBox" + idSuffix).val("");        
```

Comment:
I cherry-picked the changes I made to the test cases to master. As expected, this happened.
![expectedfail](https://cloud.githubusercontent.com/assets/7261051/6865835/0c934ab2-d4ad-11e4-8fed-c67261fb2f54.png)